dev/com.ibm.ws.security.oidc.client_fat.jaxrs/fat/src/com/ibm/ws/security/openidconnect/client/fat/jaxrs/CommonTests/JaxRSClientAPITests.java

```diff
@@ -267,7 +318,7 @@ public void APIOidcJaxRSClientTests_jaxrsOAuthClientProperty_string_false() thro
 
     }
 
-    @Mode(TestMode.LITE)
+    //        @Mode(TestMode.LITE)
```

Comment:
This intended to stay?

Reply:
no, I can clean it up when I mess with userinfo